components/lock.rst

```diff
@@ -431,6 +474,30 @@ method uses the Memcached's ``flush()`` method which purges and removes everythi
     The method ``flush()`` must not be called, or locks should be stored in a
     dedicated Memcached service away from Cache.
 
+PdoStore
+~~~~~~~~~~
+
+The PdoStore relies on the `ACID`_ properties of the SQL engine.
+
+.. caution::
+
+    In a cluster configured with multiple masters, ensure writes are
```

Comment:
Is it correct to use the word `master` (in the context master/slave database)?

ping @lsmith77 or @javiereguiluz

Reply:
master/slave is a bad terminology (see https://twitter.com/dhh/status/1032268675674296320)